lib/util.js

```diff
@@ -1590,6 +1590,23 @@ util.encode64 = function(input, maxline) {
   return output;
 };
 
+/**
+ * Encodes input according to the "Base64url Encoding" format as specified
+ * in JSON Web Signature (JWS) RFC7517. A URL safe character set is used and
+ * trailing '=', line breaks, whitespace, and other characters are omitted.
+ *
+ * @param input the data to encode.
```

Comment:
Can we specify what format `data` is in? Is it a binary string? I would assume so.

Reply:
Yep, fixing.